docs/devel/offline-virtual-machine.md

```diff
@@ -0,0 +1,234 @@
+# Offline Virtual Machine developer documentation
+
+This document introduces the offline virtual machine object and provides a
+guide how to use it and build upon it.
+
+## What is Offline Virtual Machine
+
+Almost all virtual machine (VM) management systems allow you to manage both running
+and stopped virtual machines. Such system allows you to edit configuration of
+both types of VMs and show its statuses.
+
+To allow building such VM management systems on top of the KubeVirt, the
+OfflineVirtualMachine object is introduced to provide the access to the stopped
+virtual machines. When working with running virtual machines, please see
+the [Virtual Machine object documentation](). The Virtual Machine object is
+designed to work in tandem with the OfflineVirtualMachine and provides the
+configuration and status for running virtual machines.
+
+OfflineVirtualMachine is a Kubernetes [CRD](https://kubernetes.io/docs/concepts/api-extension/custom-resources/), which
+allows for using the Kubernetes machanisms for storing the objects in the etcd,
+exposing REST API and CLI.
```

Comment:
You should probably mention the Operator pattern and declarative as well in this paragraph.

Reply:
I am not sure if it is relevant for this concrete text. But it would make a good blog post.